src/components/admin/DepartmentNote.vue

```diff
@@ -93,7 +93,7 @@
           text
           @click="onCancelSave"
         >
-          Cancel
+          Cancel{{ item ? 'Edit ' : 'Create ' }}<span class="sr-only">Note</span>
```

Comment:
@lyttam - I think we need a space following "Cancel"

Reply:
@johncrossman good catch. Amended; I also added context to the confirmation modal buttons.